src/Microsoft.OData.Client/DataServiceStreamResponse.cs

```diff
@@ -64,7 +64,7 @@ public string ContentDisposition
         }
 
         /// <summary>Gets the collection of headers from the response.</summary>
-        /// <returns>The headers collection from the response message as a <see cref="T:System.Collections.Generic.Dictionary`2" /> object.</returns>
+        /// <returns>The headers collection from the response message as a <see cref="System.Collections.Generic.Dictionary`2" /> object.</returns>

```

Comment:
I didn't resolve ALL warnings. I will split this across multiple pull requests, as otherwise it will get too big to review. There will still be warnings left after this pull request is completed. Either way, your comment is definitely valid, I added the change.